File: src/main/resources/static/js/word/word.js
/**
 *
 */
class Word {
	constructor(word, wordRome) {
		this.word = word;
		this.wordRome = wordRome;
	}

	get word() {
		return this._word;
	}

	set word(value) {
		this._word = value;
	}

	get wordRome() {
		return this._wordRome;
	}

	set wordRome(value) {
		this._wordRome = value;
	}

	getWord() {
		return this.word + ":" + this.wordRome;
	}

}

File: src/main/java/com/example/demo/domain/Word.java
package com.example.demo.domain;

import java.io.Serializable;
import java.util.Date;

import lombok.Data;

@Data
public class Word implements Serializable {

	private  final static long serialVersionUID = 1L;

	/** 単語ID */
	private String wordId;
	/** 単語 **/
	private String wordStr;
	/** 単語ローマ字 */
	private String wordRome;
	/** 登録日 */
	private Date inpDate;
	/** 更新日 */
	private Date updDate;

}
